neqo-transport/src/connection.rs

```diff
@@ -3240,6 +3305,7 @@ mod tests {
             test_fixture::DEFAULT_ALPN,
             &test_fixture::anti_replay(),
             Rc::new(RefCell::new(FixedConnectionIdManager::new(6))),
+            QuicVersion::Draft28,
         )
         .expect("create a server");
 
```

Comment:
Can you add a test here that ensures that we can connect with every version we support?  We'll need one for the server too.

Reply:
opened #692.